examples/medplum-chart-demo/src/bots/core/charting-utils.ts

```diff
@@ -277,7 +276,7 @@ export function createBundle(resources: (Condition | Observation | ClinicalImpre
   const entries: BundleEntry[] = resources.map((resource) => {
     const entry: BundleEntry = {
       fullUrl: `urn:uuid:${randomUUID()}`,
-      request: { method: 'PUT', url: getUpsertUrl(resource) },
+      request: { method: 'POST', url: resource.resourceType },
```

Comment:
I removed the upsert from the demo in order to make it work with our conservative/defensive limits on `serializable` transactions.  If the intent is just to represent an all-or-nothing create of a set of resources that couldn't possibly exist any other way, the upsert isn't really necessary